.github/workflows/generator_generic_slsa3.yml

```diff
@@ -34,6 +34,11 @@ on:
         required: false
         type: boolean
         default: false
+      attestation-name:
+        description: "The artifact name of the signed provenance. (Default: attestation.intoto.jsonl)"
```

Comment:
should we fix the extension to be intoto.jsonl?

Reply:
Good point. I'll add code to do that. Thanks